.github/workflows/pr-test.yml

```diff
@@ -0,0 +1,39 @@
+name: Test Engine Theme SDK
+
+on:
+  pull_request:
+    # synchronize	commit(s) pushed to the pull request
+    types: [synchronize, opened]
+
+jobs:
+  test_and_lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Setup Node with 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+          registry-url: "https://npm.pkg.github.com"
+
+      - name: Cache Node Modules
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('package-lock.json') }}
+
+      - name: Install dependencies if lockfile changed
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: npm install
```

Comment:
While not as important as in the fusion-news-theme-blocks repo, still think using `npm ci` is a nice safeguard to ensuring tests run accurately.

Reply:
huh interesting

Reply:
will do

Reply:
updated ✅